### DISCUSSION TITLE
Is Panda working with turbo on development in nextjs?

### DISCUSSION THREAD

**Discussion by juliomuhlbauer** [chakra-ui/panda#2456] [General]
I am using Park UI and it is showing this error: 
```
 ⨯ ./src/app/globals.css:5581:24
Parsing css source code failed
  5579 |
  5580 |   .border_border\.emphasized {
> 5581 |     border-color: border. emphasized;
       |                         ^
  5582 |   }
  5583 |
  5584 |   .self_stretch {

Expected Declaration value
```

**Comment by astahmer** [comment]
> Is Panda working with turbo on development in nextjs? #2456

yes

---

can you make a minimal repro ? your issue looks like you're missing a token

**Reply by juliomuhlbauer** [reply]
You were right. There was a token missing. Strange that it was working with webpack. Thanks!
